snap/sdc-config/sdc.conf

```diff
@@ -0,0 +1,4 @@
+{
+	"includePath": ["/snap/sdc/current/import", "."],
+	"libPath": ["/snap/sdc/current/lib"],
```

Comment:
It would be nicer to use `@SDCPATH@/../import` and `@SDCPATH@/../lib` here, but AFAICT this doesn't actually work, despite being used in `sdc.conf.buildConf()`.  Is support for the `@SDCPATH@` placeholder actually implemented yet (it doesn't seem to be) or is it intended for future implementation?